### PR TITLE
Logo Particle: Add option to disable url

### PR DESCRIPTION
**Pull request by adi8i**
If you like the approach :). #1607

### CODE REVIEW

File: engines/common/nucleus/particles/logo.yaml [hunk 1]
```diff
@@ -20,6 +20,12 @@ form:
       type: input.imagepicker
       label: Image
       description: Select desired logo image.
+      
+    url_disable:
+      type: input.checkbox
+      label: Disable Url
+      description: Disable image url.
```
Comment (w00fz) [inline engines/common/nucleus/particles/logo.yaml]: Can you change `Url` to `URL`. Or perhaps even just `No Link` fits better.
Also the description to `Renders Logo/Image without any link`

File: engines/common/nucleus/particles/logo.yaml [hunk 2]
```diff
@@ -20,6 +20,12 @@ form:
       type: input.imagepicker
       label: Image
       description: Select desired logo image.
+      
+    url_disable:
```
Comment (w00fz) [inline engines/common/nucleus/particles/logo.yaml]: Maybe `nolink` is a better naming?

**Comment by marktaylor46** [comment]
Great stuff @adi8i But I agree with @w00fz that it should be "no link" for the field label.

**Comment by w00fz** [comment]
Thanks @adi8i

**Comment by mahagr** [comment]
Umm, there's one minor thing I don't like about this change: the option being `No Link`. I never like negative options as checking a checkbox really means enabling something, not disabling it. To me its counter intuitive to enable disabling something and I think its also considered as a bad practise by usability people.

As an afterthought it would have been better if the particle would have disabled link if you had empty link field. :) But I guess we cannot change that anymore, so this change is the second best thing to do.

**Comment by marktaylor46** [comment]
@mahagr You can't do that because when the field it empty the particle uses the "home" page as the link instead i.e. there is never an option to have no link at all since blank for link actually means "home".

**Comment by adi8i** [comment]
Maybe a dropdown would be better with the following options, link to: Home page, Particle Link, No Link.

With better naming ofc.

**Comment by marktaylor46** [comment]
All I would say is you have to consider existing sites with "blank" url and those with link url completed - they must behave the same as they do now - I don't think a dropdown would give you that because you would have to default to one thing and which it was would be wrong in some cases.

**Comment by mahagr** [comment]
@marktaylor46 That's why I said its an afterthought.

I'm mostly fine with the change, its good. The only thing I don't like is to have `No Link` instead of `Enable Link`.

**Comment by mahagr** [comment]
Also it might be good to have a placeholder saying `Home Page` instead of just having empty text field, as with this new option it is a bit confusing what the meaning of empty have.

**Comment by marktaylor46** [comment]
Yah I don't really mind how it gets changed as long as it's backwards compatible with existing sites. Also "home page" was my term it actually isn't that it's "default menu item" really (which is usually home but doesn't have to be).

**Comment by adi8i** [comment]
https://github.com/adi8i/gantry5/commit/4f5740c14678ee0e991c942bafd96c2f750de683 @mahagr would be better like this ?

**Comment by mahagr** [comment]
Yeah, and then default the option to 1 as well in yaml file.

**Pull request merged**